### DISCUSSION TITLE
Afficher un type selon une validation personnalisée d'un champ de date

### DISCUSSION THREAD

**Discussion by ageliinas** [MTESSDev/FRW#137] [Annonces]
Bonjour,
J'aimerais savoir s'il est possible, et si oui comment, d'afficher (avec un v-if) un type comme un type radio par exemple selon la validation personnalisée d'un champ de date. Par exemple, si mon champ de date valide que la date saisie est inférieur à 24 mois (validation perso), on affiche ou non la question suivante. 

Merci! :)

**Comment by j0ber** [comment]
Bonjour, 
Il suffirait que le code de votre validation custom utilise une computed. Vous pourriez alors réutiliser cette computed dans votre v-if.

Cela dit, vous allez avoir le même problème de computed dans un v-if (#129). 

Il faudra donc déjà utiliser la solution de contournement.
